docs/pages/eas-update/migrate-codepush-to-eas-update.mdx

```diff
@@ -0,0 +1,99 @@
+
+Remove the CodePush SDK from your project by uninstalling the `react-native-code-push` package:
+
+<Terminal cmd={['$ npm uninstall react-native-code-push']} />
+
+You'll also need to remove CodePush references from JS and native code. Follow this [guide](https://github.com/Microsoft/react-native-code-push/issues/1101#issuecomment-350204507) for more detailed instructions.
+
+</Step>
+
+<Step label="2">
+## Ensure `app.json` exists
+
+Ensure that your project has an `app.json` file. If you don't have one, you can create one by running the command:
+
+<Terminal
+  cmd={[
+    "# If app.json doesn't exist, create a basic one",
+    '[ -f app.json ] || echo "{}" > app.json',
+  ]}
+/>
+
+</Step>
+
+<Step label="3">
+## Install the `expo` package
+
+Install the `expo` package by running the command:
+
+<Terminal cmd={['npx install-expo-modules@latest']} />
+
+If the command fails, follow the installation instructions [here](/bare/installing-expo-modules/#manual-installation)
+
+</Step>
+
+<Step label="4">
+## Turn off resource bundle signing (XCode 14+ only)
+
+In Xcode 14 and higher, resource bundles are signed by default, which requires setting the development team for each resource bundle target.
+To resolve this issue, downgrade to an older Xcode version using the `image` field in `eas.json`, or turn off signing resource bundles in your Podfile. See this [example here](https://expo.fyi/r/disable-bundle-resource-signing).
+
+</Step>
+
+<Step label="5">
+## Install the latest EAS CLI
+
+EAS CLI is the command-line app that you will use to interact with EAS services from your terminal. To install it, run the command:
+
+<Terminal cmd={['$ npm install -g eas-cli']} />
+
+You can also use the above command to check if a new version of EAS CLI is available. We encourage you to always stay up to date with the latest version.
+
+> We recommend using `npm` instead of `yarn` for global package installations. You may alternatively use `npx eas-cli`, just remember to use that instead of `eas` whenever it's called for in the documentation.
+
+</Step>
+
+<Step label="6">
+## Log in to your Expo account
+
+If you are already signed in to an Expo account using Expo CLI, you can skip the steps described in this section. If you are not, run the following command to log in:
+
+<Terminal cmd={['$ eas login']} />
+
+You can check whether you are logged in by running `eas whoami`.
+
+</Step>
+
+<Step label="7">
+## Configure project, deploy builds and beyond
+
+We have finished all the steps tailored for a CodePush-enabled bare React Native project. To proceed with the configuration of your project, please begin with Step 4 in [the main guide](https://docs.expo.dev/eas-update/getting-started/#create-a-build-for-the-project) and follow it through to completion.
```

Comment:
It would be nice if this particular migration guide ended with something more concrete and verifiable.

Reply:
Also perhaps a bit about how you will need to rebuild and submit to the store in order for users to get the new update.